phoenix-core/src/main/java/org/apache/phoenix/mapreduce/index/IndexUpgradeTool.java

```diff
@@ -235,6 +240,11 @@ public CommandLine parseOptions(String[] args) {
                     +TABLE_OPTION.getLongOpt() + " and " + TABLE_CSV_FILE_OPTION.getLongOpt()
                     + "; specify only one.");
         }
+        if ((cmdLine.hasOption(INDEX_TOOL_OPTION.getOpt()))
```

Comment:
Just a thought, would it be simpler to have just one IndexUpgradeTool option which when present indicates both do-rebuild as well as give IndexTool options? We wouldn't need this check then.

Reply:
@priyankporwal Are you suggesting something like this `-rebuild '-v ONLY'` ? I agree that would simplify some things. The only thing is that a value will always have to be passed to that option even if you want to run IndexTool with default options. Options can either always take a value or never take a value.

Reply:
Yeah, that's what I had in mind, but I see the need to have some value for that option would an empty string in quotes. Anyways, current solution works too.